### DISCUSSION TITLE
access it again

### DISCUSSION THREAD

**Discussion by mbauomi** [TheLastBen/fast-stable-diffusion#954] [Q&A] [answered]
I already did all of the processes and got the trained model but I can't reaccess it after I spent a while inactive on the colab page so is there a way to reaccess the trained model or do I need to go through the whole process again

**Comment by TheLastBen** [comment, marked as answer]
in the session cell, enter the name of the previous session and it will load it

**Reply by mbauomi** [reply]
I did try to do that but it show a box that tells me that I should enter the name of the model when I do that It says there is no model with that name.

**Reply by TheLastBen** [reply]
that's the testing cell, there is a "Create/Load Session" cell above
